### ISSUE TITLE
Generating tests only in uppercase letters

### ISSUE THREAD

**Issue by zehranaz** [anithag/kleestr#4]
In the strcat tests, all the test cases generated are always in uppercase letters even though to be a valid case, it should be in lowercase. Is this a translation issue (i.e. going from CVC4 to Klee, the written tests just aren't properly formatted), or is this a bigger problem?

**Comment by anithag** [comment]
Yes, noticed this while testing strlen. I haven't debugged but AFAIK its not a CVC4 problem.
